### PR TITLE
Shorten mining progress logs.

### PR DESCRIPTION
**Pull request by jean-m-cyr**
- No need to count stale rejects separately.
- Use minimal spacing... line gets very long for multi GPU
- Other than accepted, only display non zero solution stats.

### CODE REVIEW

File: libethcore/Miner.h [hunk 1]
```diff
@@ -123,10 +120,9 @@ inline std::ostream& operator<<(std::ostream& _out, WorkingProgress _p)
 			}
 		}
 
-		_out << "gpu/" << i << " " << EthTeal << std::fixed << std::setw(5) << std::setprecision(2) << mh << EthReset;
+		_out << " gpu/" << i << " " << EthTeal << std::fixed << std::setprecision(2) << mh << EthReset;
```
Comment (chfast) [inline libethcore/Miner.h]: Someone suggested to numerate GPUs as `gpu0` not `gpu/0`. I don't mind.

**Comment by AndreaLanfranchi** [comment]
Is it possible to get rid **completely** of the rejectedStale counter ?
It has no meaning at all IMHO

**Comment by jean-m-cyr** [comment]
@AndreaLanfranchi Already got rid of it!

File: libethcore/Miner.h (hunk 1 above)
Comment (jean-m-cyr) [reply libethcore/Miner.h]: Done

**Pull request merged**